# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/xf86-video-impact/hotfix-resrange.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- xf86-video-impact/src/impact_driver.c.vanilla	2024-04-09 23:25:03.051797285 +0200
+++ xf86-video-impact/src/impact_driver.c	2024-04-09 23:26:37.336796872 +0200
@@ -249,12 +248,8 @@
 					entity = xf86ClaimNoSlot(drv, 0, dev, TRUE);
 					if (probedIDs[j].sr) {
 						base = (IMPACTSR_BASE_ADDR0 + busID * IMPACTSR_BASE_OFFSET);
-						RANGE(range[0], base,
-							base + sizeof(ImpactSRRegs), ResExcMemBlock);
 					} else {
 						base = (IMPACTI2_BASE_ADDR0 + busID * IMPACTI2_BASE_OFFSET);
-						RANGE(range[0], base,
-							base + sizeof(ImpactI2Regs), ResExcMemBlock);
 					}
 					pScrn = xf86AllocateScreen(drv, 0);
 					xf86AddEntityToScreen(pScrn, entity);
